kyuubi-server/src/main/scala/org/apache/kyuubi/engine/JpsApplicationOperation.scala

```diff
@@ -65,6 +65,9 @@ class JpsApplicationOperation extends ApplicationOperation {
       val (id, _) = idAndCmd.splitAt(idAndCmd.indexOf(" "))
       try {
         s"kill -15 $id".lineStream
+        if (getEngine(tag).nonEmpty) {
+          killApplicationByTag(tag)
```

Comment:
to prevent flaky test.

Reply:
spark-submit will generate two processes.

Reply:
if so the first kill will fail since there is no that process, shall we check the result of the command ? if the process is not found we should try to kill again, otherwise do nothing

Reply:
sounds good, I thought that the command will not throw exception even it failed to kill the process.

Just have a testing, it will throw exception if the command fail.

done.